### ISSUE TITLE
optional arguments for fortios_system_sdnconnector

### ISSUE THREAD

**Issue by mbdraks** [fortinetdev/terraform-provider-fortios#85]
Arguments ha_status, server, username, password should be optional.

If type is **aws** those arguments are not relevant and the CLI does not even show them.

The current workaround is to send an empty string.

```terraform
resource "fortios_system_sdnconnector" "this" {
  name             = "aws_sdn"
  status           = "enable"
  type             = "aws"
  use_metadata_iam = "disable"
  access_key       = aws_iam_access_key.this.id
  secret_key       = aws_iam_access_key.this.secret
  region           = data.aws_region.this.name
  vpc_id           = aws_vpc.this.id
  ha_status        = ""
  server           = ""
  username         = ""
  password         = ""
}
```

**Comment by frankshen01** [comment]
Hi @mbdraks , Barbosa, thanks for your correction. It has been fixed and will be released in the next version(this week).

**Comment by frankshen01** [comment]
Fixed and released, please see the latest version(v1.6.1): https://registry.terraform.io/providers/fortinetdev/fortios/latest, Thanks!
